tests/Unit/inc/classes/subscriber/FacebookTrackingCacheBustingSubscriber/deleteCache.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit\inc\classes\subscriber\FacebookTrackingCacheBustingSubscriber;
+
+use Mockery;
+use WP_Rocket\Admin\Options_Data;
+use WP_Rocket\Busting\Busting_Factory;
+use WP_Rocket\Busting\Facebook_Pickles;
+use WP_Rocket\Busting\Facebook_DSK;
```

Comment:
```suggestion
use WP_Rocket\Busting\Facebook_SDK;
```

I almost thought you were making a joke with this one 😆

Reply:
As I expected, you solved my "undefined class" issue :heart: